components/dashboard/PortfolioSummary.tsx

```diff
@@ -12,6 +12,11 @@ import ClaimModal from "../discover/claimModal";
 import SuccessModal from "../discover/successModal";
 import { useHidePortfolio } from "@hooks/useHidePortfolio";
 
+// Format the numbers with commas
+const formatNumberWithCommas = (number: number): string => {

```

Comment:
Good but you need to add that to `numberService.ts` and test the function with jest in `numberService.test.js`